changelogs/fragments/4175-opentelemetry_logs.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - opentelemetry callback plugin - send logs (https://github.com/ansible-collections/community.general/pull/4175).
```

Comment:
```suggestion
  - opentelemetry callback plugin - send logs. This can be disabled by setting ``disable_logs=false`` (https://github.com/ansible-collections/community.general/pull/4175).
```